crates/bevy_render/src/color/mod.rs

```diff
@@ -406,6 +408,237 @@ impl Color {
         self
     }
 
+    // Linear RGB
+
+    /// Get the red in linear rgb colorspace.
+    pub fn linear_rgb_r(&self) -> f32 {
```

Comment:
For consistency with how the color space name is used elsewhere, could you rename this (and the others) to the form of `rgb_linear_*`?